configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
+			CACert string `yaml:"ca,omitempty"`
```

Comment:
Probably something to do with certs only being signed by a CA. 

But I guess one could accept different groups of certs. I'm unsure what the advantage would be though.